src/database/migrations/20180423122353_create-tokens.js

```diff
@@ -18,6 +18,10 @@ export const up = async knex => {
       .timestamp('createdAt')
       .notNull()
       .defaultTo(knex.fn.now())
+    table
+      .timestamp('updatedAt')
+      .notNull()
+      .defaultTo(knex.fn.now())

```

Comment:
If `updatedAt` has to have a valu by default, let’s edit all migrations for consistency then.